core/include/thread.h

```diff
@@ -200,7 +200,8 @@ struct _thread {
  *
  * Creating a new thread is done in two steps:
  * 1. the new thread's stack is initialized depending on the platform
- * 2. the new thread is added to the scheduler to be run
+ * 2. the new thread is added to the scheduler and the scheduler is run (if not
```

Comment:
Depends on what the scheduler decides when it runs, i.e. is the new thread's priority higher than the currently running one's.